generators/heroku/index.js

```diff
@@ -26,6 +27,8 @@ const statistics = require('../statistics');
 
 const constants = require('../generator-constants');
 
+const execCmd = util.promisify(exec);
```

Comment:
See https://nodejs.org/api/child_process.html. As I did not change everything to async/await I decided to have dedicated command which is "promisified"